### PR TITLE
some fixes to dockerfile and docker-compose.yml

### PR DESCRIPTION
**Pull request by granludo**
now works on osx
using mariadb

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,9 @@ FROM php:8.1-apache
 #Install git
 EXPOSE 80
 # Modules that this needs
-RUN apt-get update && apt-get install -y git netcat && apt-get clean -y
+# RUN apt-get update && apt-get install -y git netcat && apt-get clean -y
+RUN apt-get update && apt-get install -y git netcat-openbsd && apt-get clean -y
```
Comment (jonespm) [inline Dockerfile:6]: This is fine, I think this was split between `netcat-traditional` and `netcat-openbsd` and either should be fine here.

File: docker-compose.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ version: '3'
 services:
   db:
     container_name: tsugi_db
-    image: mysql:5.7
+    image: mariadb:10.5
```
Comment (jonespm) [inline docker-compose.yml:8]: I think it's up to @csev on whether we have mariadb or mysql:8.0 here. I'm fine with maria.

**Comment by jonespm** [comment]
I haven't tested this may also need to get updated to php:8.2 to be compatible with master

**Comment by csev** [comment]
@granludo I agree with @jonespm - I don't generally test or use this - so how about when you go to PHP 8.3 you send in another PR for that.

**Pull request merged**